From 98d79a4ea5acc38550b1cead1dc0e876c710273f Mon Sep 17 00:00:00 2001
From: Nicolas Dufresne <nicolas.dufresne@collabora.com>
Date: Thu, 2 Nov 2023 10:41:18 -0400
Subject: [PATCH 53/68] v4l2codecs: format: Prefer mplane variants

If we have the choice, prefer the mplane variant of the pixel formats
since it allows importing the single plane variant.
Upstream-Status: Pending
---
 sys/v4l2codecs/gstv4l2format.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2codecs/gstv4l2format.c b/sys/v4l2codecs/gstv4l2format.c
index e029d7d..7af79f9 100644
--- a/sys/v4l2codecs/gstv4l2format.c
+++ b/sys/v4l2codecs/gstv4l2format.c
@@ -32,8 +32,8 @@ struct FormatEntry
 };
 
 static struct FormatEntry format_map[] = {
-  {V4L2_PIX_FMT_NV12, 1, GST_VIDEO_FORMAT_NV12, 8, 420},
   {V4L2_PIX_FMT_NV12M, 2, GST_VIDEO_FORMAT_NV12, 8, 420},
+  {V4L2_PIX_FMT_NV12, 1, GST_VIDEO_FORMAT_NV12, 8, 420},
   {V4L2_PIX_FMT_YUYV, 1, GST_VIDEO_FORMAT_YUY2, 8, 422},
   {V4L2_PIX_FMT_SUNXI_TILED_NV12, 1, GST_VIDEO_FORMAT_NV12_32L32, 8, 422},
   {V4L2_PIX_FMT_NV12_4L4, 1, GST_VIDEO_FORMAT_NV12_4L4, 8, 420},
-- 
2.25.1

